build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Yeah I had the debate of comment / pure JSON. I'll add the comment. Pretty much everything supprots comments in json.